grid_generation/Gen_CICE_grid.py

```diff
@@ -0,0 +1,96 @@
+"""
+Script: generate_cice_grid.py
```

Comment:
The name here does not match the actual file name. I would suggest renaming the file to `generate_cice_grid.py`.

Reply:
Thanks @micaeljtoliveira. I have updated the filenames in this commit  8cd8480